acme/acme/crypto_util_test.py

```diff
@@ -151,6 +151,55 @@ def test_sn_collisions(self):
             self.serial_num.append(cert.get_serial_number())
         self.assertTrue(len(set(self.serial_num)) > 1)
 
+class FunctionTest(unittest.TestCase):
```

Comment:
If `make_private_key` is removed we can change this class to `TestMakeCSR` to match the convention followed in most of the rest of the repo where a test class tests a single function/class/etc.